src/app/components/avatar/avatar.css

```diff
@@ -0,0 +1,29 @@
+.avatar img {
+  border-radius: 50%;
+  /* margin-left: 0.5rem; */
+}
+
+.avatar--xl img {
+  height: 9rem;
+  width: 9rem;
+}
```

Comment:
in this case giving height and width directly to the image will stretch the image if the provided img resolution is not square-shaped.

Reply:
Done pankaj.